org.eclipse.wildwebdeveloper/package.json

```diff
@@ -9,6 +9,7 @@
     "yaml-language-server": "1.14.0",
     "vscode-languageserver-protocol": "3.17.4-next.3",
     "vscode-languageserver-types": "3.17.4-next.2",
+    "vscode-languageserver": "9.0.0",
     "vscode-css-languageserver": "file:target/vscode-css-languageserver-1.0.0.tgz",
```

Comment:
After this vscode-languageserver is in root so nested node_modules is not required

Reply:
I downgraded to 8.2 (9.0 have just 5 days)